### DISCUSSION TITLE
Have a special 'traitor perk tree' for anyone assigned as a traitor

### DISCUSSION THREAD

**Discussion by FarrunDragon** [FakeFishGames/Barotrauma#7650] [Ideas and suggestions]
I think it would be cool to have a perk tree that is added on the side of the other (so you can be both engineer and traitor) that would have minor bonuses to nefarious deeds; later perks would give you the chance to slip handcuffs (failure causes injury) and earlier perks might just help you be more aware of where people are, grow eggs faster, etc

**Comment by StruggleDuck** [comment]
this is actually a really neat idea, id love to see something like this added.

**Reply by StruggleDuck** [reply]
to add on, I think that depending on what your traitor objective is, you could have some items spawn in your inventory as well, that way you dont get gunned down by security because you need to get a weapon to assassinate your target with.
